### PR TITLE
Dsg 2595/align nuverial text select icons

### PR DESCRIPTION
**Pull request by sunnyjiao-nv**
# Basics

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Styling and layout changes
- [ ] Code style changes (formatting, local variables)
- [ ] Refactoring (no functional changes)
- [ ] Build or CI related changes
- [ ] Tests
- [ ] Other... Please describe:

## What changes does this PR introduce?

Adjust Nuverial text input & select input margins to align, and more closely match Figma designs

## Screenshot of the affected changes
Before:
e78ec9d6-70d6-4033-aca2-40741655a340
After:
Screenshot 2024-01-11 at 1 32 04 PM

## Checklist

- [ ] I have updated the documentation accordingly
- [ ] I have confirmed all acceptance criteria defined by the story have been met
- [ ] I have confirmed that Sonar Qube findings for new code are reviewed and fixed, or will be whitelisted

### CODE REVIEW

File: libs/shared/ui/nuverial/src/lib/components/select/select.component.html
Comment (sunnyjiao-nv) [inline libs/shared/ui/nuverial/src/lib/components/select/select.component.html:1]: matPrefix was removed from nuverial-icons inside select options, as it was conflicting with styling of the field's prefix icon

File: libs/shared/ui/nuverial/src/lib/components/select/select.component.html [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+  <nuverial-button *ngIf="prefixIcon" buttonStyle="icon-no-border" matPrefix>
+    <nuverial-icon [iconName]="prefixIcon"></nuverial-icon>
+  </nuverial-button>
 
   <nuverial-icon
+    class="selectedOptionPrefixIcon"
     *ngIf="selectedOption?.prefixIcon"
     [iconName]="selectedOption?.prefixIcon"
     [style.--nuverial-icon-color]="selectedOption?.color"
-    matPrefix
```
Comment (sunnyjiao-nv) [inline libs/shared/ui/nuverial/src/lib/components/select/select.component.html:9]: matPrefix was removed from nuverial-icons inside select options, as it was conflicting with styling of the field's prefix icon

**Review by thechechogomez — CHANGES_REQUESTED** [review]
https://github.com/Nuvalence/dsgov-web/assets/60438413/a5d40c96-8efa-4981-95bf-f3602d60dce5

Looks good. Just a small thing. Can we add an `align-items: center` on the nuverial button on the suffix for the select.

**Review by thechechogomez — APPROVED** [review]
LGTM

**Pull request merged**